pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -33,6 +36,11 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+const (
+	appName                   = "rook-ceph-mgr-external"
```

Comment:
In this package we wouldn't want other components accidentally be named with this appName
```suggestion
	externalMgrAppName                   = "rook-ceph-mgr-external"
```